### PR TITLE
Redesign versions

### PR DESCRIPTION
**Pull request by MarcelGeo**
**Description**

Scope: versions

image

- Version detail sidebar

image

- version table primevue
- all version changsets tables to primevue

image

- breacrumps component to alpha with router configuration in beforeEnter for /projects paths
- sidebar active class updates based on figma

:up: Refactor:
- Implemented enums based on modules. When adding new view to the modules/project/views directory, it's recommended to create corresponding route names for this view. These route names can be used for routing purposes within the library or in other applications.

:exclamation: 
- router updates for breadcrumps

### CODE REVIEW

File: web-app/packages/lib/src/modules/project/views/ProjectVersionsViewTemplate.vue [hunk 1]
```diff
@@ -292,6 +300,24 @@ export default defineComponent({
         ...v,
         disabled: this.disabledKeys.some((d) => d === v.name)
       }))
+    },
+    filterMenuItems(): MenuItem[] {
+      return [
+        {
+          label: 'Newest versions',
+          key: 'newst',
```
Comment (varmar05) [inline web-app/packages/lib/src/modules/project/views/ProjectVersionsViewTemplate.vue:308]: typo?

File: web-app/packages/app/src/router.ts [hunk 2]
```diff
@@ -146,6 +149,21 @@ export const createRouter = (pinia: Pinia) => {
           return
         }
       },
+      /** Redirect of unused /history/:version_id path to /history?version_id */
```
Comment (varmar05) [inline web-app/packages/app/src/router.ts:152]: where is this used?

File: web-app/packages/lib/src/modules/layout/components/SideBarTemplate.vue [hunk 3]
```diff
@@ -63,20 +63,20 @@ import { useLayoutStore } from '@/modules/layout/store'
...
     {
-      active: currentPage.value === 'dashboard',
+      active: route.matched.some((item) => item.name === 'dashboard'),
       title: 'Dashboard',
       to: '/dashboard',
       icon: 'ti ti-home'
     },
     {
-      active: currentPage.value === 'projects',
+      // TODO: hardcoded path names add to enum
```
Comment (varmar05) [inline web-app/packages/lib/src/modules/layout/components/SideBarTemplate.vue:76]: please resolve

File: web-app/packages/app/src/router.ts (hunk 2 above)
Comment (MarcelGeo) [reply web-app/packages/app/src/router.ts:152]: compatibility with links

File: web-app/packages/lib/src/modules/layout/components/SideBarTemplate.vue (hunk 3 above)
Comment (MarcelGeo) [reply web-app/packages/lib/src/modules/layout/components/SideBarTemplate.vue]: Implemented enums based on modules. When adding new view to the modules/project/views directory, it's recommended to create corresponding route names for this view. These route names can be used for routing purposes within the library or in other applications.

**Pull request merged**